### DISCUSSION TITLE
Add alternative keybindings for Home, End, Delete, PageUp, PageDown keys

### DISCUSSION THREAD

**Discussion by emilia-miki** [helix-editor/helix#9957] [General] [answered]
Many keyboards, especially on laptops, do not have Home/End/Delete/PageUp/PageDown keys, and these keys are hard to reach on other keyboards. I suggest to add alternative default keybindings for actions binded to these keys. 
This is especially relevant to Picker, as it currently can't be remapped.

**Comment by sicher** [comment, marked as answer]
They are already bound:

ctrl-b - pageup
ctrl-f - pagedown
gh - start of line
gl - end of line

**Comment by kirawi** [comment]
Those keybindings are included only for legacy compatibility. They have alternate keys (see https://docs.helix-editor.com/keymap.html). As for picker rebinding, see https://github.com/helix-editor/helix/issues/5505

**Reply by emilia-miki** [reply]
The docs don't mention alternate keys for Normal Mode goto_line_start and goto_line_end. I'll suggest an edit.

**Reply by emilia-miki** [reply]
https://github.com/helix-editor/helix/pull/9961

**Comment by alexeymuranov** [comment]
Related: #9896.
